src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -185,10 +168,10 @@ export const validationSchema = Yup.object()
     showReactions: Yup.bool().notRequired(),
     auditoriumColumns: Yup.number()
       .notRequired()
-      .min(5, "Columns must be at least 5"),
+      .min(5, "There must be at least 5 columns"),
     auditoriumRows: Yup.number()
       .notRequired()
-      .min(5, "Rows must be at least 5"),
+      .min(5, "There must be at least 5 rows"),
```

Comment:
Is this enforced/handled somehow in the auditorium (`Audience` component) at all? And if so, is it using any `const`s extracted into `src/settings.ts`?

We want to extract the `5`'s here into `src/settings.ts`, and use the same `const`s used in `Audience`.

Reply:
I don't think so. There isn't a variable in `settings.ts` that relates to this and in `Audience` the minimum values are hard coded as 25 and 19 (so we should change it to either 5x5 or 25x19). Should I implement this?

Reply:
@malaikaaiyar If I remember correctly, I think the values set in `Audience` might be the defaults, but not the minimum allowed. If that's the case, then this should just be extracted as new `const`s.

---

@mike-lvov @denisdimitrov Do you know what we should be setting here?

Reply:
>  If I remember correctly, I think the values set in Audience might be the defaults, but not the minimum allowed. If that's the case, then this should just be extracted as new consts.


That's correct. Those values are the default ones. 

> Do you know what we should be setting here?

Nope. We've never talked about the min/max values for this venue template. Maybe we can set it to 5x5 or just ask product team for some input.

Reply:
@sparkletown/product Do you have any thoughts for the minimum size the auditorium should be allowed to be set to? Currently `5`.

Reply:
@0xdevalias there are no tech minimums neither for rows nor for columns.

Reply:
There is now a settings variable called MINIMUM_COLUMNS and MAXIMUM_COLUMNS. Can I use this for both the rows and columns?

Reply:
fixed in 2f465aa7a6c887c40d3e054ac5059ab66d0e2a33